### ISSUE TITLE
Move relative objects to Otter.Colliders namespace

### ISSUE THREAD

**Issue by WillSams** [WillSams/Otter#4]
In an effort to move everything out of the general "Otter" namespace, all objects in .cs files under the Otter/Colliders should be moved to Otter.Colliders namespace.

No specs (tests) need to be written for this task, just clean & rebuild project and run FlippyFlop to test if the changes did not cause an adverse impact.

If you want to contribute:

1. Make a fork of the repository.
2. Switch to the test-otter-epic branch and do a git pull.
3. Make your changes and commit.
4. Submit a pull request.

Once the pull request is approved, I'll merge your code into the test-otter-epic branch. Code from test-otter-epic branch will be merged into dotnetcore-dev once all stories within the epic are completed.

**Comment by WillSams** [comment]
Finis.
